ui/ozone/platform/wayland/wayland_pointer.cc

```diff
@@ -104,6 +100,7 @@ void WaylandPointer::Button(void* data,
   EventType type;
   if (state == WL_POINTER_BUTTON_STATE_PRESSED) {
     type = ET_MOUSE_PRESSED;
+    pointer->SetSerial(serial);
```

Comment:
Maybe adding a comment here about why we are doing it here and not anywhere else.